### DISCUSSION TITLE
Git Submodule Support

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#699] [Feature Requests]
submodules.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by edrex** [comment]
Another possibility is to use gittrees, let a bot handle committing+pushing subtree changes on push. Even if this is implemented in the app, consider using subtrees instead of or in addition, as they are less fraught with peril.
